subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -1259,6 +1277,18 @@ int nrf_cloud_pgps_process_update(uint8_t *buf, size_t len)
 	size_t need;
 	int64_t gps_sec;
 
+	/* Note:
```

Comment:
should this note be in the function header comment instead?

Reply:
yeah, good idea